include/quill/LogMacros.h

```diff
@@ -580,6 +581,51 @@
   #define QUILL_LOGJ_INFO_TAGS(logger, tags, fmt, ...) (void)0
 #endif
 
+#if QUILL_COMPILE_ACTIVE_LOG_LEVEL <= QUILL_COMPILE_ACTIVE_LOG_LEVEL_NOTICE
+  #define QUILL_LOG_NOTICE(logger, fmt, ...)                                                         \
+    QUILL_LOGGER_CALL(QUILL_LIKELY, logger, quill::LogLevel::Notice, fmt, ##__VA_ARGS__)
+
+  #define QUILL_LOG_NOTICE_LIMIT(min_interval, logger, fmt, ...)                                     \
+    QUILL_LOGGER_CALL_LIMIT(min_interval, QUILL_LIKELY, logger, quill::LogLevel::Notice, fmt, ##__VA_ARGS__)
+
+  #define QUILL_LOG_NOTICE_WITH_TAGS(logger, tags, fmt, ...)                                         \
+    QUILL_LOGGER_CALL_WITH_TAGS(QUILL_LIKELY, logger, quill::LogLevel::Notice, tags, fmt, ##__VA_ARGS__)
```

Comment:
This has been replaced on master with QUILL_LOGGER_CALL and you pass either nullptr or the tags as one of the arguments.

the similar was done for the LIMIT macros